src/sentry/web/frontend/project_settings.py

```diff
@@ -186,9 +205,12 @@ def handle(self, request, organization, team, project):
                     'scrub_data',
                     'scrub_defaults',
                     'sensitive_fields',
-                    'scrub_ip_addresses',
+                    'scrub_ip_address',
```

Comment:
There are tests over a different endpoint, the API endpoint, which doesn't even accept this setting as a value. I'm adding some more tests around this stuff.